index.bs

```diff
@@ -656,13 +690,21 @@ To <dfn>obtain an attribution trigger</dfn> given a [=URL=] |url| and an
     :: |triggerPriority|.
 1. Return |trigger|.
 
-<dfn>Max trigger data value</dfn> is a vendor-specific integer which controls the potential values of [=attribution trigger/trigger data=].
+<dfn>Trigger data cardinality</dfn> is a vendor-specific integer that controls the valid range of
+[=attribution trigger/trigger data=]:
+0 <= [=attribution trigger/trigger data=] < [=trigger data cardinality=].
 
-<dfn>Max event source trigger data value</dfn> is a vendor-specific integer which controls the potential values of [=attribution trigger/event source trigger data=].
+<dfn>Event-source trigger data cardinality</dfn> is a vendor-specific integer that controls the valid
+range of [=attribution trigger/event source trigger data=]:
+0 <= [=attribution trigger/event source trigger data=] < [=event-source trigger data cardinality=].
 
-<dfn>Trigger data noise rate</dfn> is a vendor-specific double between 0 and 1 (both inclusive) which controls noising of [=attribution trigger/trigger data=].
+<dfn>False trigger data probability</dfn> is a vendor-specific double between 0 and 1 (both
```

Comment:
We should name these definitions to align with how we'd expose them externally in the API. I don't think they are quite right currently "false trigger data" implies this will always return something that is not correct but this is not always the case - there is a chance that the randomized output matches the true value by luck.

Some ideas:
- Randomized trigger data {probability, rate}
- Trigger data randomized response rate
- Trigger data random output rate
- etc

cc @johnivdel

Reply:
Renamed to `randomized trigger data rate`.